File: src/b1happynumber/Go/main.go
// main.go

package main

import (
	"fmt"
	"strings"
)

func main() {
	inputs := []int{1, 5, 19, 25, 7}
	for i, input := range inputs {
		fmt.Printf("%d.\tInput Number: %d\n", i+1, input)

		result := "False"
		if isHappyNumber(input) {
			result = "True"
		}

		fmt.Printf("\n\tIs it a happy number? %s\n", result)
		fmt.Println("-" + strings.Repeat("-", 99))
	}
}

File: src/b1happynumber/Python/main.py
from happy_number_helper import is_happy_number


def main():
    inputs = [1, 5, 19, 25, 7]
    for input_num in inputs:
        print(f"Input Number: {input_num}")
        print(f"Is it a happy number? {is_happy_number(input_num)}")
        print("-" * 50)


if __name__ == "__main__":
    main()
